client/components/cards/attachments.js

```diff
@@ -20,6 +20,33 @@ Template.attachmentsGalery.events({
   'click .js-remove-cover'() {
     Cards.findOne(this.cardId).unsetCover();
   },
+  'click .js-preview-image': Popup.open('previewAttachedImage'),
+});
+
+Template.previewAttachedImagePopup.onRendered(function() {
+  // when there are multiple thumbnails,
+  // if click one then another very fast, we will get a wrong width value.
+  // adding a little delay will avoid the bug.
```

Comment:
Could you try removing the `setTimeout` and executing `Tracker.flush()` before measuring the image to see if it works in the edge case you are mentioning?